docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -84,9 +84,11 @@ The name should be just the name, without any additional info. For example, don'
 - Apostrophe (Special Edition)
   :::
 
+If the app is a distinct edition among different versions of your app on Flathub (e.g. `Plex` for desktop versus `Plex HTPC` for the home theater version), then using the full name including the edition may be acceptable.
+
 ### No weird formatting
 
-The name should not have any weird formatting or punctuation. For example, it should not be all-lowercase, all-uppercase, camel case, or contain dashes or periods. Cases where the weird formatting is part of the brand are exempt from this (e.g. `VLC`).
+The name should not have any weird formatting or punctuation. For example, it should not be all-lowercase, all-uppercase, camel case, pascal case, or contain dashes or periods. Cases where the formatting is part of an established brand may be exempt (e.g. `VLC`).
 
 :::danger Bad examples
```

Comment:
Can you also add an example of a bad camel case name?

Reply:
I added pascal case because someone pedantically pointed out we meant that instead of camel case; I'm fine to just not include camel case in the list. :)

Reply:
I'm basically saying that I think pascal case is fine. But tobias probably disagrees.

Reply:
Or maybe not.

TestApp is okay from a casing standpoint, but QTApp is not. Both are pascal case. I guess we have other rules to catch these.

Reply:
> TestApp is okay from a casing standpoint

Yeah I disagree, I think any of the below are problematic:

- newsflash
- newsFlash
- NewsFlash

What people should do instead:

- News Flash
- Newsflash

Reply:
- `DuckStation` as a wordplay for PlayStation, wouldn't work the other way

Recognized brands:
- `LibreOffice` (could argue it's a recognized brand
- `RetroArch` also recognized brand
- `FreeTube`

Reply:
This whole conversation makes me wonder if we can/should have "reviewer notes" on why a decision was made one way or the other, to document this (and so we have a sort of log of precedent; it would be annoying for DuckStation if one reviewer said the name was fine, then another reviewer said it wasn't, later).

That's far outside the scope of this docs update, though; I think we will need to subjectively decide on edge cases no matter what the docs say.

Reply:
We have a database field for this, it's just not exposed

Reply:
In general, we don't want to be too strict and we can be some more lenient depending on cases. Icon sizes come to mind for e.g. we're not looking for a 100% size match.

But the guidelines should be as clear as possible and probably ignore that.